cmd/pipeline/delete.go

```diff
@@ -80,13 +79,11 @@ func deletePipeline(cmd *cobra.Command, options DeleteOptions) error {
 	resp, err := gateClient.PipelineControllerApi.DeletePipelineUsingDELETE(gateClient.Context, options.application, options.name)
 
 	if err != nil {
-		util.UI.Error(fmt.Sprintf("%s\n", err))
```

Comment:
Are there any calls to `UI.Error` left? Can we remove that function as well?

Reply:
yes, in gateway client and savePipeline

Reply:
UI.Error use in gateway client and savePipeline